src/main/resources/index.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
```

Comment:
Because of:
<img width="1741" alt="Screenshot 2022-08-15 at 09 57 21" src="https://user-images.githubusercontent.com/47667812/184590112-67a14e87-980f-4a06-b1a2-84fc65df71ba.png">